mcs/class/System.Private.CoreLib/Makefile

```diff
@@ -9,7 +9,7 @@ dirs := $(dir $(wildcard */*))
 files := $(wildcard */*.cs)
 
 all-local: $(ROSLYN_PROPS_FILE)
-	dotnet build $(CORETARGETS) -p:BuildArch=$(COREARCH) -p:OutputPath=bin/$(COREARCH) -p:RoslynPropsFile="$(ROSLYN_PROPS_FILE)" System.Private.CoreLib.csproj
+	dotnet build $(CORETARGETS) $(CORLIB_BUILD_FLAGS) -p:BuildArch=$(COREARCH) -p:OutputPath=bin/$(COREARCH) -p:RoslynPropsFile="$(ROSLYN_PROPS_FILE)" System.Private.CoreLib.csproj
```

Comment:
for additional arguments, e.g. (from netcore folder) `make bcl CORLIB_BUILD_FLAGS="-c Debug"`